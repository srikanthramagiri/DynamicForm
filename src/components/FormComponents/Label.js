import React from "react";
import { Form } from "react-bootstrap";
import PropTypes from "prop-types";

export default function Label(props) {
  let {  label } = props;
  return <Form.Label >{label }</Form.Label>;
}

Label.propTypes = {
  name: PropTypes.string,
  label: PropTypes.string,
};
