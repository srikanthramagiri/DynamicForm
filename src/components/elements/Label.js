import React from "react";
import { Form } from "react-bootstrap";

export default function Label(props) {
  let { name, label } = props;
  return <Form.Label htmlFor={name}>{label}</Form.Label>;
}
