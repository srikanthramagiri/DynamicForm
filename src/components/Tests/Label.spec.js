import { shallow } from 'enzyme';
import Label from '../FormComponents/Label'

describe('Label component', ()=> {
    it('should render correctly', ()=> {
        const wrapper = shallow(<Label />)
        expect(wrapper).toMatchSnapshot();
    })
})