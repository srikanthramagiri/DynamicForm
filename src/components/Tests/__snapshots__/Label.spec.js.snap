// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AlertError component should render correctly 1`] = `ShallowWrapper {}`;

exports[`AlertError should render correctly 1`] = `ShallowWrapper {}`;

exports[`Label component should render correctly 1`] = `ShallowWrapper {}`;
